modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -406,6 +640,25 @@ export default function EditNotificationTemplate({
 	);
 }
 
+interface Item {
+	label?: string;
+	value?: string;
+}
+
+interface Role {
+	accountId: number;
+	description: string;
+	displayName: string;
+	id: number;
+	name: string;
+	roleId: number;
+}
+
+interface User {
+	alternateName: string;
+	givenName: string;
+}
+
```

Comment:
Hey @alinedoleron, could you move all these `type` declarations and `interfaces` to the top of the file? We are following a new standard, so each `interface` or `type` you see at the bottom of the file you can move to the top now :smile: